SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -98,9 +98,9 @@ private static void PlatformNotifyUserConfirmation(UserConfirmation userConfirma
             Instance.InstanceHandlerUserConfirmation(userConfirmation);
         }
 
-        private static void PlatformTrackError(System.Exception exception, IDictionary<string, string> properties)
+        private static void PlatformTrackError(System.Exception exception, IDictionary<string, string> properties, ErrorAttachmentLog[] attachments)
```

Comment:
Have you changed this signature for other platforms?

Reply:
@jaelim-ms are you talking about Xamarin Android/iOS? That is in another PR

Reply:
This breaks CI

Reply:
CI should be fixed once #1171  is merged.